rfc/cdevents-spinnaker.md

```diff
@@ -0,0 +1,161 @@
+# Implementing CDEvents to Spinnaker using Java SDK
+
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | _#299_ |
+| **Author(s)**  | _Jalander Ramagiri (`@rjalander`)_|
+| **SIG / WG**   | CDF-CDEvents, CDF-SIGEvents |
+
+## Overview
+
+Many organisations use more than one CI/CD tools as part of their pipelines and most of the times the organisations implement their own glue code in order to make the tools work with each other.
+
+Implementing [CDEvents](https://cdevents.dev/) to Spinnaker enables Spinnaker to communicate with other CI/CD tools with in the ecosystem. [CDEvents](https://cdevents.dev/) is a common specification for Continuous Delivery events based on CloudEvents.
+Spinnaker can produce and consume [CDEvents](https://cdevents.dev/) using [Java SDK](https://github.com/cdevents/sdk-java) which is developed and maintained by the [CDEvents community](https://cdevents.dev/community/).
+
+## Goals and Non-Goals
+Goals:
```

Comment:
I'm not so familiar with Spinnaker, so I don't know if Spinnaker already has a mechanism to associate incoming events with the execution of tasks in Spinnaker. 

If not, from my experience working on POCs with other tools I would say:
- first, implement producing events: this is usually simpler to do and it introduces benefits in terms of observability. Events can be collected and stored to derive devops metrics about number of deployments etc
- second, implement the ability to receive events: this can be more complex because you will need a way to allow users to associate incoming events with things that should happen in Spinnaker. 

If this exists in Spinnaker already, then receiving events shouldn't be more complex then sending :)